firmware/v5.ld

```diff
@@ -28,6 +28,10 @@ MEMORY
    HOT_MEMORY : ORIGIN = start_of_hot_mem, LENGTH = _HOT_MEM_SIZE  /* Just over 8 MB */
 }
 
+/* used in _sbrk implementation */
+_HEAP_START = ORIGIN(HEAP);
+_HEAP_END = ORIGIN(HEAP) + LENGTH(HEAP);
+
 REGION_ALIAS("RAM", COLD_MEMORY);
 
-ENTRY(vexStartup)
+ENTRY(main)
```

Comment:
I would avoid taking up the `main` symbol here. I think CRTs typically use `_start` or similar.